drag_and_drop_v2/templates/html/drag_and_drop_edit.html

```diff
@@ -86,21 +117,29 @@
             </section>
             <section class="tab-content">
                 <form class="display-labels-form">
-                    <h3>{% trans "Zone labels" %}</h3>
-                    <label for="display-labels">{% trans "Display label names on the image" %}:</label>
-                    <input name="display-labels" id="display-labels" type="checkbox" />
+                    <h4>{% trans "Zone labels" %}</h4>
+                    <label class="checkbox-label">
+                        <input name="display-labels" class="display-labels" type="checkbox" />
+                        <span>{% trans "Display label names on the image" %}</span>
+                    </label>
                 </form>
                 <form class="display-borders-form">
-                    <h3>{% trans "Zone borders" %}</h3>
-                    <label for="display-borders">{% trans "Display zone borders on the image" %}:</label>
-                    <input name="display-borders" id="display-borders" type="checkbox" />
+                    <h4>{% trans "Zone borders" %}</h4>
+                    <label class="checkbox-label">
+                        <input name="display-borders" class="display-borders" type="checkbox" />
+                        <span>{% trans "Display zone borders on the image" %}</span>
+                    </label>
                 </form>
             </section>
             <section class="tab-content">
+                <h4>{% trans "Zone definitions" %}</h4>
                 <div class="zone-editor">
                     <div class="controls">
                         <form class="zones-form"></form>
-                        <a href="#" class="add-zone add-element"><div class="icon add"></div>{% trans "Add a zone" %}</a>
+                        <button class="btn add-zone add-element">
+                            <span class="icon add"></span>
```

Comment:
I'm not sure what type of icon this is, but for Font Awesome icons we always specify aria-hidden (with either visible text like this or screen reader text).

See example HTML here--
http://ux.edx.org/design_elements/icons/